config/prow/cluster/monitoring/mixins/lib/config.libsonnet

```diff
@@ -1,80 +1,77 @@
-      {instance: "104.197.27.114:9090", type: "scalability-project", friendly: "Scalability project"},
-      {instance: "35.225.208.117:9090", type: "scalability-project", friendly: "Scalability project (k8s-infra)"},
-      {instance: "104.197.27.114:9090", type: "scalability-presubmit-project", friendly: "Scalability presubmit project"}
-    ],
+  // Tide pools that are important enough to have their own graphs on the dashboard.
+  tideDashboardExplicitPools: [
+    {org: 'kubernetes', repo: 'kubernetes', branch: 'master'},
+  ],
 
-    // How long we go during work hours without seeing a webhook before alerting.
-    webhookMissingAlertInterval: '10m',
+  // Additional scraping endpoints
+  probeTargets: [
+  # ATTENTION: Keep this in sync with the list in ../../additional-scrape-configs_secret.yaml
+    {url: 'https://prow.k8s.io', labels: {slo: comps.deck}},
+    {url: 'https://monitoring.prow.k8s.io', labels: {slo: comps.monitoring}},
+    {url: 'https://testgrid.k8s.io', labels: {}},
+    {url: 'https://gubernator.k8s.io', labels: {}},
+    {url: 'https://gubernator.k8s.io/pr/fejta', labels: {}}, # Deep health check of someone's PR dashboard.
+    {url: 'https://storage.googleapis.com/k8s-gubernator/triage/index.html', labels: {}},
+    {url: 'https://storage.googleapis.com/test-infra-oncall/oncall.html', labels: {}},
+  ],
 
-    // How many days prow hasn't been bumped.
-    prowImageStaleByDays: 7,
-  },
+  // Boskos endpoints to be monitored
+  boskosResourcetypes: [
+    {instance: "104.197.27.114:9090", type: "aws-account", friendly: "AWS account"},
+    {instance: "104.197.27.114:9090", type: "gce-project", friendly: "GCE project"},
+    {instance: "35.225.208.117:9090", type: "gce-project", friendly: "GCE project (k8s-infra)"},
+    {instance: "104.197.27.114:9090", type: "gke-project", friendly: "GKE project"},
+    {instance: "104.197.27.114:9090", type: "gpu-project", friendly: "GPU project"},
+    {instance: "35.225.208.117:9090", type: "gpu-project", friendly: "GPU project (k8s-infra)"},
+    {instance: "104.197.27.114:9090", type: "ingress-project", friendly: "Ingress project"},
+    {instance: "104.197.27.114:9090", type: "node-e2e-project", friendly: "Node e2e project"},
+    {instance: "104.197.27.114:9090", type: "scalability-project", friendly: "Scalability project"},
+    {instance: "35.225.208.117:9090", type: "scalability-project", friendly: "Scalability project (k8s-infra)"},
+    {instance: "104.197.27.114:9090", type: "scalability-presubmit-project", friendly: "Scalability presubmit project"}
+  ],
+
+  // How long we go during work hours without seeing a webhook before alerting.
+  webhookMissingAlertInterval: '10m',
+
+  // How many days prow hasn't been bumped.
+  prowImageStaleByDays: 7,
+};
+
+// Generate the real config by adding in constant fields and defaulting where needed.
+{
+  _config+:: util.defaultConfig(util.consts + config),
```

Comment:
Nit, this seems a bit excessive to me to call method from another module with the value from the other module, is there a way to make `defaultConfig` always use `consts`?

Reply:
Yeah thats part of the reason I moved the constants to the new file. I forgot to change this. Fixed.